superbuild/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:18.04
```

Comment:
For information 18.04 is on EOL since last month, should we consider using 20.04?

Reply:
lets wait for potential user to update for a while